src/main/kotlin/app/revanced/patches/music/misc/microg/patch/bytecode/MusicMicroGBytecodePatch.kt

```diff
@@ -0,0 +1,28 @@
+package app.revanced.patches.music.misc.microg.patch.bytecode
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.implementation.BytecodeData
+import app.revanced.patcher.patch.annotations.Dependencies
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.patch.implementation.BytecodePatch
+import app.revanced.patcher.patch.implementation.misc.PatchResult
+import app.revanced.patcher.patch.implementation.misc.PatchResultSuccess
+import app.revanced.patches.music.misc.microg.annotations.MusicMicroGPatchCompatibility
+import app.revanced.patches.youtube.misc.microg.patch.resource.MicroGResourcePatch
+
+@Patch(include = false)
+@Dependencies(dependencies = [MicroGResourcePatch::class])
+@Name("music-microg-support")
```

Comment:
```suggestion
@Name("microg-support")
```
Consistency with the other MicroG Support patch. The manager filters this correctly, not sure about the CLI. If not, that should be fixed on the CLI.

Reply:
The cli uses the patch name as the key to add the patch, it has to be distinct

Reply:
> The manager filters this correctly, not sure about the CLI. If not, that should be fixed on the CLI.

As I said above, that needs to be properly filtered on the CLI. So that's a CLI issue.